substrate/frame/contracts/fixtures/build.rs

```diff
@@ -254,7 +254,7 @@ fn invoke_riscv_build(current_dir: &Path) -> Result<()> {
 		.expect("failed to execute process");
 
 	if build_res.status.success() {
-		return Ok(())
+		return Ok(());
```

Comment:
Is this enforced by the new formatter?  
Otherwise please revert..

Reply:
Sorry, vscode decided to add them, fixed